class.jetpack.php

```diff
@@ -378,6 +378,14 @@ class Jetpack {
 	 */
 	public static $plugin_upgrade_lock_key = 'jetpack_upgrade_lock';
 
+	/**
+	 * The highest action hook priority that was used for the plugins_loaded event.
+	 * Used in the `add_configure_hook' method.
+	 *
+	 * @var Integer the priority value.
+	 */
+	protected $highest_priority = null;
```

Comment:
Just a nitpick, but what do you think of using a different variable name here? We also have the local `$highest_priority` variable in the `add_configure_hook()` method. The two variables have the same name but represent different things.

Maybe something like `$configure_hook_priority`?

Reply:
You're right, thanks, I'll change that!